nucypher/cli/main.py

```diff
@@ -19,8 +19,8 @@
 
 import click
 
-from nucypher.cli import status, stake
-from nucypher.cli.characters import moe, ursula, alice, bob, enrico, felix
+from nucypher.cli import status
+from nucypher.cli.characters import moe, ursula, alice, bob, enrico, felix, stake
```

Comment:
Related to @tuxxy comment above: `stake` is not a character. But that's a discussion beyond this PR.